library/src/SPSApplication.ts

```diff
@@ -61,7 +61,7 @@ export class SPSApplication extends Application {
 		this.stream.setSignallingUrlBuilder(() => {
 
 			// if we have overriden the signalling server URL with a .env file use it here
-			if (WEBSOCKET_URL) {
+			if (WEBSOCKET_URL != "") {
```

Comment:
This will fail in prod where `WEBSOCKET_URL` is being set as `undefined`:

```typescript
plugins: [
    new webpack.DefinePlugin({
      WEBSOCKET_URL: undefined
    }),
  ]
```

Reply:
this has been moved to https://github.com/ScalablePixelStreaming/Frontend/pull/44 PR